src/main/java/tasks/Todo.java

```diff
@@ -0,0 +1,24 @@
+package tasks;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import tasks.Task;
+
+
+/**
+ * The Todo class represents a Todo.
+ */
+@JsonTypeName("tasks.Todo")
+public class Todo extends Task {
+
+
+    public static final String taskType = "T";
```

Comment:
Perhaps you can use the `SCREAMING_SNAKE_CASE` for your constants (also applies all your related `Task` subclasses), as in `TASK_TYPE`? I understand that Java's CheckStyle might think otherwise though...